$NetBSD: patch-doc_api_Makefile_am,v 1.1 2012/08/18 15:13:51 dsainty Exp $

Doxygen produces man pages for the build directories too, which we really
don't want to install.  These are tricky to match, but are the man pages
that start and finish with a single underscore character.  Prune them out.

--- doc/api/Makefile.am.orig	2008-11-07 10:42:52.000000000 +1300
+++ doc/api/Makefile.am	2012-08-19 02:12:34.354453680 +1200
@@ -230,6 +230,7 @@
 	$(mkinstalldirs) $(DOC_INST_DIR)/man/man3
 	@list='$(wildcard man/man3/*.3)'; \
 	for file in $$list ; do \
+	  case "$$file" in */_[!_]*[!_]_.3) continue;; esac; \
 	  echo " $(INSTALL_DATA) $$file $(DOC_INST_DIR)/$$file"; \
 	  $(INSTALL_DATA) $$file $(DOC_INST_DIR)/$$file; \
 	done
